packages/stepper/src/Stepper.story.tsx

```diff
@@ -26,26 +24,45 @@ storiesOf('Stepper', module)
     </div>
   ))
   .add('Many', () => {
-    let currentStep = number('Step', 0, { min: 0 });
-    const numSteps = number('Number of steps', 7, { min: 1 });
+    let currentStep = number('Step', 1, { min: 0 });
+    const numSteps = number('Number of steps', 10, { min: 1 });
+    const maxDisplayedSteps = number('Max displayed', 5, { min: 1 });
+    let completedStepsShown = number('Completed steps shown', 3, { min: 1 });
+    const darkMode = boolean('Dark mode', false);
 
     // Can't dynamically change the max, so we manually enforce it
-    if (currentStep + 1 > numSteps) {
+    if (currentStep >= numSteps) {
       channel.emit(CHANGE, {
         name: 'Step',
         value: numSteps - 1,
       });
       currentStep = numSteps - 1;
     }
 
+    // Can't dynamically change the max, so we manually enforce it
+    if (completedStepsShown >= maxDisplayedSteps - 2) {
+      channel.emit(CHANGE, {
+        name: 'Completed steps shown',
+        value: maxDisplayedSteps - 2,
+      });
+      completedStepsShown = maxDisplayedSteps - 2;
+    }
```

Comment:
That only happens when there's more than 1 step, which shouldn't really happen. The line displaying is a minor bug I can fix, but I think we should support `maxDisplayedSteps = 1` because it renders like this when there's just one step:

![image](https://user-images.githubusercontent.com/99368875/163850946-d0623fe3-68a4-4cd6-8e09-47197064c459.png)